library/command/command/src/main/java/org/quiltmc/qsl/command/impl/KnownArgumentTypesSync.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019, 2020 zml and Colonel contributors
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.command.impl;
+
+import java.util.HashSet;
+
+import io.netty.buffer.Unpooled;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import org.jetbrains.annotations.ApiStatus;
+
+import net.minecraft.network.PacketByteBuf;
+import net.minecraft.util.Identifier;
+
+import org.quiltmc.qsl.networking.api.ServerPlayNetworking;
+import org.quiltmc.qsl.networking.api.client.C2SPlayChannelEvents;
+
+@ApiStatus.Internal
+public final class KnownArgumentTypesSync {
```

Comment:
This can probably be done with an entry point event, removing the extra `if (loaded) ...`

Reply:
Oh, right, forgot about Aurora's new innovations.

Reply:
Actually, wait, the server side can't be done with entrypoints since it isn't actually an event.

Reply:
![image](https://user-images.githubusercontent.com/7997354/152642569-9106a2cd-d16a-4eb6-bc83-6abc3f3fde14.png)